integration/time_drift/time_drift.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will make sure that guest time
+# is synced with the host
+
+set -e
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Save guest time
+GUEST_TMP_FILE=$(mktemp guest.XXXXX)
+# Save host time
+HOST_TMP_FILE=$(mktemp host.XXXXX)
+
+function remove_tmp_files() {
+	rm "${GUEST_TMP_FILE}" "${HOST_TMP_FILE}"
+}
+
+trap remove_tmp_files EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "Verify guest time is synced with the host" {
+	CONTAINER_NAME="test"
+	IMAGE="ubuntu"
+	PAYLOAD="tail -f /dev/null"
+
+	docker run -d --name "${CONTAINER_NAME}" --runtime "${RUNTIME}" "${IMAGE}" sh -c "${PAYLOAD}"
+
+	# Get host and guest time
+	(docker exec "${CONTAINER_NAME}" date > "${GUEST_TMP_FILE}"; date > "${HOST_TMP_FILE}") | parallel
```

Comment:
I think maybe this is not using `parallel` as you expect? I think you'd either need to echo the commands at `parallel`, or put them on its command line, no?